modules/ROOT/pages/troubleshooting.adoc

```diff
@@ -138,3 +138,7 @@ If guest users do not see published API versions that have been set to be visibl
 . Open Salesforce setup.
 . Open the sharing rules.
 . Ensure that the sharing rule for the object `CommunityApi` matches your community name exactly, including spacing and letter case.
+
+== Refresh the Cache
+
+If changes made to APIs are not yet visible, you can refresh the cache to see them.
```

Comment:
here we need to explain how to get to the cache config screen and click on the "Clear cache" button:

1. In your salesforce organization, go to Setup > Platform Cache
2. Open ACMCache partition
3. Click on Clear Cache button

Reply:
add also:
Observe that if your caches are not currently being used, you will not see the Clear Cache button.

Reply:
Added. Thank you!